src/get-started/java-binding.md

```diff
@@ -290,11 +290,11 @@ Transactions are transmitted by external service clients to the framework as
 [Exonum messages][transactions-messages].
 A transaction message contains:
 
-- a header with the identifying information, such as a numeric ID
```

Comment:
Nit: AFAIK, the first letter of a list item is not capitalized after a semicolon (only after a dot).